src/Raven.Server/RavenServerStartup.cs

```diff
@@ -218,11 +287,31 @@ private async Task RequestHandler(HttpContext context)
                 }
             }
         }
+        /// <summary>
+        /// IsQuery method checks if watched URL contains "?query" OR "multi_get" OR "index-fields"
+        /// </summary>
+        /// <param name="context"></param>
+        /// <returns>boolean</returns>
+        private static bool IsQuery(HttpContext context)
+        {
+            return (
+                    context.Request.QueryString.Value.StartsWith("?query") 
```

Comment:
We need to watch everything. Not only those three.

Reply:
all urls are currently watched, those three actions currently return content to the customInfo field

Reply:
Yes, but we should not limit ourselfs. If there is a body and we have a traffic-watch turned on, then we should track the body and show it in the UI

Reply:
@ppekrol I'm not sure that we want to just get the body of all requests in this manner.
In particular, `bulk_docs` and `BulkInsert` are two scenarios that we don't want to capture in this way.
Too many details that would overwhelm the user.

Reply:
Body can be very large, and we don't usually want to show the raw data.
For queries / patch is pretty much the only case.
bulk_docs isn't, we don't want the full docs there.